packages/react/README.md

```diff
@@ -34,7 +35,7 @@ yarn add carbon-components-react carbon-components carbon-icons
    [`carbon-components-react/.storybook/polyfills.js` file](./.storybook/polyfills.js)
    is required.
 
-If you just want to try out `carbon-components-react`, you can also use
+If you want to try out `carbon-components-react`, you can also use
```

Comment:
```suggestion
If you only want to try out `carbon-components-react`, you can also use
```

Maybe this would be the equivalent?

Reply:
Thank you! Just committed this suggestion 👍